tests/scalers/metrics_api/metrics_api_test.go

```diff
@@ -161,6 +161,8 @@ spec:
     metadata:
       targetValue: "5"
       activationTargetValue: "20"
+      ignoreUnavailable: "true"
+      defaultValue: "0"
```

Comment:
This isn't enough to test the behaviour, please add a test case (inside the same file is enough) to cover this new feature.